### DISCUSSION TITLE
FYI: Too little memory for F2FS

### DISCUSSION THREAD

**Discussion by solskogen** [MiSTerArch/PKGBUILDs#15] [General]
The 512M of memory available for Linux is to little for F2FS. If you try you will get a lot of errors in dmesg, and some operations will fail, like git clone.

**Comment by amstan** [comment]
Sigh, that's ridiculous. BTRFS works with even compression enabled but F2FS fails. I wonder what's going on.

Are there any mount options that could perhaps make this work?

Can you paste some pieces of the dmesg you're seeing. I can't seem to find any other people on google with the same problem.

**Comment by solskogen** [comment]
I've deleted the contents of that sd card now, but there's a lot of messages in dmesg about f2fs and memory. I had to try cloning the Distribution_MiSTer at least 4 times before it didn't fail. That was with zstd for compression. Don't think any fstab option would help :( I tried even getting a swapfile on it, but that also failed (it should work!)

**Comment by amstan** [comment]
You might want to try without compression, just the default options.

You might be getting into weird untested territory by doing zstd all the time.

**Comment by solskogen** [comment]
That might be true. It's just that zstd is the overall fastest for both compression and decompression on :)
